src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js

```diff
@@ -217,6 +232,28 @@ class PopoverReportActionContextMenu extends React.Component {
         });
     }
 
+    /**
+     * Used to calculate the Context Menu Dimensions
+     *
+     * @returns {JSX}
+     */
+    measureContent() {
+        return (
+            <BaseReportActionContextMenu
+                type={this.state.type}
+                isVisible
+                selection={this.state.selection}
+                reportID={this.state.reportID}
+                reportAction={this.state.reportAction}
+                isArchivedRoom={this.state.isArchivedRoom}
+                isChronosReport={this.state.isChronosReport}
+                isPinnedChat={this.state.isPinnedChat}
+                anchor={this.contextMenuTargetNode}
+                contentRef={this.setContentRef}
+            />
+        );
+    }
+
```

Comment:
```suggestion
```

This looks like leftover from resolving conflicts when merging main, since I see that was removed [here](https://github.com/Expensify/App/pull/18883/files#diff-16a590a45460a9abf1e402441b50e3cb2659ee3da203ca30a2fa4ce3f9411957L47)